meson.build

```diff
@@ -47,9 +43,20 @@ endif
 
 frigg_dep = declare_dependency(include_directories: incl)
 
+if not get_option('build_tests').disabled()
```

Comment:
Isn't this outer if statement redundant?

Reply:
yes, making a mental note to re check when not half asleep

Reply:
LGTM otherwise